web/openapi/openapi.yaml

```diff
@@ -16,6 +16,73 @@ components:
       in: header
       name: X-VAST-Token
 paths:
+  /query:
```

Comment:
For symmetry with the `/query/<id>` endpoint, should `/query` maybe also take `n` and return `position`? I imagine that it will make things easier as a user (of the backend directly and for the frontend implementation).

Reply:
IIRC our discussions, the idea is that `/query` serves as "constructor" returning the ID, so it's the endpoint to use when you don't have a query yet. So `n` and `position` are not defined at this point.

Reply:
I just noticed that I actually forget to commit the latest changes before opening the PR, locally I had already renamed these to `/query/new` and `/query/:id:/next` because I also felt the endpoint names were not very intuitive.

Since they also contain a few bug fixes I'm going to push them, and if you like the original names better feel free to open another comment to revert the renaming @mavam